cfpb/cfgov-refresh
# Move Director's Notebook category from Newsroom to Blog

Addresses https://GHE/CFGOV/platform/issues/3789

## Changes

- Move Director's Notebook category from Newsroom to Blog

## Testing

1. Pull branch
1. Migrate
1. Create a new blog post, giving it the "Director's notebook" category
1. Ensure it displays properly on the blog listing page and that you can filter by that category

## Notes

- Blog post categories are currently not showing up on individual blog post pages. An issue has been opened to potentially address this: https://GHE/CFGOV/platform/issues/3790

## Checklist
